### DISCUSSION TITLE
Yarn run app doesn't launch dev server

### DISCUSSION THREAD

**Discussion by Codinards** [vercel/next.js#56606] [Help]
### Summary

I have created a next app. But when I run **yarn run dev** command, dev server is not started and I just recieve the output below:
![image](https://github.com/vercel/next.js/assets/50141117/c0a07e85-605a-4188-9bb1-8146c59291e6)
node version: 18.18.0
next version: 13.5.4
yarn version: 1.22.5
npm version: 7.20.5

**Comment by icyJoseph** [comment]
I think this sometimes happens if your Node architecture doesn't match the OS'

Try running:

```console
node -p 'process.arch'
```

That should match your OS architecture.
